ooiui/static/js/views/common/SVGView.js

```diff
@@ -690,10 +689,16 @@ var SVGPlotControlView = Backbone.View.extend({
 
 
       this.$start_date.datetimepicker({defaultDate : moment(this.model.get('start')),
-                                       maxDate: moment(this.model.get('end'))
+                                       minDate: moment(this.model.get('start')),
```

Comment:
Don't need utc there. the utc part sets _isUTC: True but doesn't convert the date and hence has no effect on the minDate setting. So we should be good there.